src/app/environments/environment.prod.ts

```diff
@@ -3,4 +3,6 @@ import { EnvironmentInterface } from './environment.interface';
 export const environment: EnvironmentInterface = {
   apiUrl: '',
   bedingungsbaumBaseUrl: 'https://bedingungsbaum.hochfrequenz.de',
+  auth0Domain: 'hochfrequenz.eu.auth0.com',
```

Comment:
Bitte die custom domain aus auth0 verwenden.
Wir verwenden erst damit unser Login Window Template aus Auth0 welches die Checkbox für die eMail-Zustimmung beinhaltet.

```suggestion
  auth0Domain: 'auth.hochfrequenz.de',
```